src/impl_constructors.rs

```diff
@@ -101,6 +101,30 @@ impl<S, A> ArrayBase<S, Ix1>
     {
         Self::from_vec(to_vec(linspace::range(start, end, step)))
     }
+
+    /// Create a one-dimensional array from the inclusive interval `[start,
+    /// end]` with `n` elements logarithmically spaced. `A` must be a floating
+    /// point type.
+    ///
+    /// **Panics** if `n` is greater than `isize::MAX`.
+    ///
+    /// ```rust
+    /// use ndarray::{Array, arr1};
+    ///
+    /// let array = Array::logspace(1e0, 1e3, 4);
+    /// println!("{}", array);
```

Comment:
I think we should remove the println lines (both here and below) from the doctest - you are showing what is inside the array with the assertion.